### ISSUE TITLE
sdk config of includedirs

### ISSUE THREAD

**Issue by shaqtsui** [xmake-io/xmake#3425]
### Xmake Version

2.7.6

### Operating System Version and Architecture

Ubuntu 20.04

### Describe Bug

Currently sdk manual config only support `includedirs`. As this is sdk level config, I think it makes more sense to have `sysincludedirs` instead of `includedirs`. Correct me if I am wrong here. 
If above declare make sense. Can we have `sysincludedirs` supported?

### Expected Behavior

Suppport ` set_config("sysincludedirs", sys_include_path)` 

### Project Configuration

N.A.

**Comment by waruqi** [comment]
add_includedirs

**Comment by shaqtsui** [comment]
I know `add_includedirs/add_sysincludedirs`, but I think this is target(project?) level.

**Comment by waruqi** [comment]
if you set it in target scope, it's target level.
if you set it in root scope, it's project level for all targets

**Comment by shaqtsui** [comment]
?So add_includedirs is duplicated with set_config("includedirs", "xx") ?

**Comment by waruqi** [comment]
> ?So add_includedirs is duplicated with set_config("includedirs", "xx") ?

no, set_config("includedirs", "xx") is equivalent to `xmake f --includedirs=xx` and although it also sets includedirs globally, if you configure it in xmake.lua, you should use add_includedirs whenever possible

**Comment by shaqtsui** [comment]
I am a bit confused. Correct me if I am wrong:
1, xmake support multiple config level.
2, `--includedirs` or `set_config()` is sdk level
3, `add_includedirs` is project level when put under project
4, `add_includedirs` is target level when put under target

My understanding is that sdk level supposed to be system include, while project/target level config supposed to be user include.

**Comment by waruqi** [comment]
- user configs (options, --includedirs, set_config())
- platform/toolchains level
  - project level, add_includeidirs (root scope in xmake.lua)
    - target level, add_includeidirs (target scope in xmake.lua)
